src/UI/Component/Divider/Factory.php

```diff
@@ -35,5 +35,28 @@
 	 * @return  \ILIAS\UI\Component\Divider\Horizontal
 	 */
 	public function horizontal();
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *       A Vertical Divider is used to mark a thematic change in a sequence of
+	 *       elements that are stacked from left to right.
+	 *   composition: >
+	 *     Vertical Dividers consists of a glyph-like character.
```

Comment:
This is fine for me for the moment, however, I am not sure if we need a more flexible solution in future.